projects/core/components/dialog/dialog.style.less

```diff
@@ -52,6 +56,14 @@
         border-radius: 0;
         border: none;
         padding: 48px ~'calc(50vw - 720px / 2)';
+        background: var(--tui-base-01);
+
+        .dialog-host {
+            flex: 1 1 auto;
+            display: inherit;
+            flex-direction: inherit;
```

Comment:
Do we need these 3 rules?

Reply:
Hmm I spend some time yesterday on fixing something and that is why I added those things. It might just be by shuffling some code around that is became useless. Well spotted, thanks!

Reply:
Well.. if they are not there you have this

https://user-images.githubusercontent.com/4103756/106953117-4cd53900-6732-11eb-94bc-b75a29b96edc.mp4